nautobot/core/checks.py

```diff
@@ -24,6 +26,11 @@
     id="nautobot.core.E004",
 )
 
+E005 = Error(
+    "'social_django' must be present in INSTALLED_APPS if 'social_core.backends' is in 'AUTHENTICATION_BACKENDS'",
+    id="nautobot.core.E004",
+)
+
```

Comment:
In keeping w/ Django convention, please just keep incrementing the id numbers from the last one. So this one should be `E006`. Please go ahead and just put it inline after `W005`, too.

Also, don't forget to update the `id` value.

Reply:
Also just to nitpick the error for consistency, please remove the quotes around AUTHENTICATION_BACKENDS. Thank you.